auth/jwt/transport.go

```diff
@@ -0,0 +1,89 @@
+package jwt
+
+import (
+	"fmt"
+	stdhttp "net/http"
+	"strings"
+
+	"github.com/go-kit/kit/transport/grpc"
+	"github.com/go-kit/kit/transport/http"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	BEARER        string = "bearer"
+	BEARER_FORMAT string = "Bearer %s"
+)
+
+// moves JWT token from request header to context
+// particularly useful for servers
+func ToHTTPContext() http.RequestFunc {
```

Comment:
Method comment should be of the form "ToHTTPContext moves the JWT token..." and be proper sentences.